Makefile

```diff
@@ -1,83 +1,190 @@
-include ./config.mk
+include ./config/.env
 
-TAGNAME ?= default
+TAGNAME := $(if $(TAGNAME),$(TAGNAME),distros)
 
 .PHONY: test-env-up
 test-env-up:
-	docker build . -q -f ./scripts/Dockerfile.build -t acceptance-test-${TAGNAME}
+	@docker build . -q -f ./scripts/Dockerfile.build -t acceptance-test-${TAGNAME}
 
-.PHONY: test-run
 test-run:
-	@docker run -d --name acceptance-test-${IMGNAME} -t \
-      -e AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}" \
-      -e AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}" \
-      -v ${ACCESS_KEY_LOCAL}:/go/src/github.com/rancher/distros-test-framework/config/.ssh/aws_key.pem \
-      acceptance-test-${TAGNAME} sh -c 'cd ./entrypoint; \
-    if [ -n "${TESTDIR}" ]; then \
-        if [ "${TESTDIR}" = "upgradecluster" ]; then \
-            go test -timeout=45m -v ./upgradecluster/... -installVersionOrCommit "${INSTALLTYPE}"; \
-        elif [ "${TESTDIR}" = "versionbump" ]; then \
-            go test -timeout=45m -v -tags=versionbump ./versionbump/... -cmd "${CMD}" -expectedValue "${VALUE}" \
-            -expectedValueUpgrade "${VALUEUPGRADED}" -installVersionOrCommit "${INSTALLTYPE}" -channel "${CHANNEL}" -testCase "${TESTCASE}" \
-            -deployWorkload "${DEPLOYWORKLOAD}" -workloadName "${WORKLOADNAME}" -description "${DESCRIPTION}"; \
-		elif [ "${TESTDIR}" = "mixedoscluster" ]; then \
-            go test -timeout=45m -v -tags=mixedos ./mixedoscluster/...; \
-        fi; \
-    elif [ -z "${TESTDIR}" ]; then \
-        go test -timeout=45m -v ./createcluster/...; \
-    fi;'
+	@if [ -z "$${IMGNAME}" ]; then IMGNAME=${IMGNAME}; fi; \
+	if [ -z "$${TAGNAME}" ]; then TAGNAME=${TAGNAME}; fi; \
+	  docker run -dt --name acceptance-test-$${IMGNAME} \
```

Comment:
Is this type or do we need literal $ here?

Reply:
typo, fixed